src/improvements/tasks/eth/001-opcm-upgrade-v200/config.toml

```diff
@@ -0,0 +1,27 @@
+l2chains = [
+    {name = "Base", chainId = 8453}
+]
+
+templateName = "OPCMUpgradeV200"
+
+[opcmUpgrades]
+absolutePrestates = [
+    {absolutePrestate = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9", chainId = 8453},
+]
+
+[addresses]
+OPCM = "0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76"
+StandardValidatorV200 = "0x58b758395be69ada78fe46f9653ff838f1e85251"
+ChildSafe1 = "0x9855054731540A48b28990B63DcF4f33d8AE46A1"
+
+[stateOverrides]
+# SuperchainConfig op-contracts/v2.0.0 implementation address set. This task assumes upgrade controller has already called OPCM upgrade.
+# https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml#L41
+0x95703e0982140D16f8ebA6d158FccEde42f04a4C = [
```

Comment:
These state overrides will need to eventually be removed before executing. Once the mainnet op,ink+soneium task is executed.

Reply:
Lets remove the entire state overrides section here before merging. We should merge this after execution of the 000-opcm-upgrade-v200 task execution today